jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -50,15 +50,25 @@ private Path checkAliasPath()
     {
         Path abs = path;
 
+        // Boolean to track if Path is a ZipFS based
+        boolean isZipFs = uri.toASCIIString().startsWith("jar:file:");
```

Comment:
If zipFS is to have exceptions and/or different handling, then let's create a `ZipPathResource' class and remove these conditionals from the `PathResource` class

Reply:
Nah, not needed.
I got rid of that test with the latest code.